### ISSUE TITLE
"Rúv" and "Ríkisútvarpið" returned no results

### ISSUE THREAD

**Issue by busla** [vthorsteinsson/Reynir#52]
A search for "rúv" and "ríkisútvarpið" returned no results.

That noun is quite common in Icelandic media.

<img width="1256" alt="screen shot 2017-03-18 at 21 23 37" src="https://cloud.githubusercontent.com/assets/3162968/24076112/3545538a-0c21-11e7-97c5-a92817e2af24.png">

<img width="1255" alt="screen shot 2017-03-18 at 21 21 43" src="https://cloud.githubusercontent.com/assets/3162968/24076105/1aa4621e-0c21-11e7-9ecd-64b972f84374.png">

**Comment by vthorsteinsson** [comment]
Yes, this is a known bug/missing feature. The search is sentence-based, not word/noun based. So if you type in something like "Ríkisútvarpið, RÚV, er fjölmiðill allra landsmanna" you will get relevant answers.
